mulighetsrommet-arena-ords-proxy/src/main/kotlin/no/nav/mulighetsrommet/arena_ords_proxy/Application.kt

```diff
@@ -35,13 +35,15 @@ fun initializeServer(config: Config) {
 }
 
 fun Application.configure(appConfig: AppConfig) {
+    val arenaOrdsClient = ArenaOrdsClient(appConfig.ords)
```

Comment:
Dette har jeg fikset i den branchen jeg sitter på nå. Men dette burde jeg jo ha fikset før jeg merget. Tror jeg var litt for kjapp på triggern siden jeg lente meg på grønne build checks, men den er jo ikke opprettet enda :smile: